src/Conf.ml

```diff
@@ -488,7 +488,14 @@ let info =
          should ignore. Each line in an $(b,.ocamlformat-ignore) file \
          specifies a filename relative to the directory containing the \
          $(b,.ocamlformat-ignore) file. Lines starting with $(b,#) are \
-         ignored and can be used as comments." ]
+         ignored and can be used as comments."
+    ; `P
+        "An $(b,.ocamlformat-enable) file specifies files that OCamlFormat \
+         should format even when the $(b,disable) option is set. Each line \
+         in an $(b,.ocamlformat-enable) file specifies a filename relative \
+         to the directory containing the $(b,.ocamlformat-enable) file. \
+         Lines starting with $(b,#) are ignored and can be used as \
+         comments." ]
```

Comment:
Which has precedence if a file is specified in both `.ocamlformat-enable` and `.ocamlformat-ignore`?

Reply:
With this implementation, `.ocamlformat-enable` is only checked when ocamlformat is disabled (`disable=true`), and `.ocamlformat-ignore` is only checked when ocamlformat is enabled (`disable=false`).

Reply:
That sounds reasonable -- would be good to define that in the manual pages.

Reply:
done